obal/data/playbooks/update.yml

```diff
@@ -55,3 +48,27 @@
       with_items: "{{ new_source_urls.stdout_lines | list }}"
       when:
         - "'://' not in item"
+
+    - name: get spec file
+      find:
+        pattern: "*.spec"
+        path: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}"
+      register: spec_file
+
+    - name: "Set spec file path, scl options and releasers"
+      set_fact:
+        spec_file_path: "{{ spec_file.files[0].path }}"
+
+    - name: 'Get updated version from spec'
+      command: "rpmspec --query --queryformat=%{version} --srpm {{ spec_file_path }}"
+      register: updated_version
+
+    - name: 'Create git branch'
+      command: "git checkout -b update-{{ inventory_hostname }}-{{ updated_version.stdout }}"
```

Comment:
I'm not a fan of doing git operations. In foreman-packaging I prefer to use `rpm/` and `deb/` prefixes. If obal becomes capable of doing both it can easily conflict as well. Perhaps we can have a branch prefix variable that's empty by default but set in foreman-packaging's manifest?

Reply:
I could base it off current branch and include that in the branch name.

Reply:
I think I prefer explicit configuration over autodetection in this case for simplicity, but maybe the autodetection isn't that hard to write.